doc/how-to/microcloud-with-terraform.md

```diff
@@ -0,0 +1,62 @@
+# Single Node Microcloud deployment (Nested VMs)
```

Comment:
Isn't it a multi node MicroCloud deployment at the end? 

To account for the fact that all VMs use the same underlying host maybe the title can be `Nested MicroCloud with Terraform`?